npm/pkg/controlplane/goalstateprocessor/goalstateprocessor.go

```diff
@@ -141,11 +150,12 @@ func (gsp *GoalStateProcessor) processIPSetsApplyEvent(goalState *protos.GoalSta
 		case ipsets.ListSet:
 			applyErr = gsp.applyLists(ipset, cachedIPSet)
 		case ipsets.UnknownKind:
-			applyErr = fmt.Errorf("Unknown IPSet kind %s", cachedIPSet.Kind)
-		}
-		if applyErr != nil {
-			return applyErr
+			applyErr = npmerrors.SimpleErrorWrapper(
+				"failed to decode IPSet apply event",
+				fmt.Errorf("Unknown IPSet kind %s", cachedIPSet.Kind),
```

Comment:
lint things: why wrap a new error? Can fix this lint with `npmerrors.SimpleError(fmt.Sprintf())`